README.rst

```diff
@@ -39,7 +39,7 @@ Example
         y_vv = Y_rv.clone()
         initial_state = nuts.new_state(y_vv, logprob_fn)
 
-        next_step = kernel(*initial_state, 1e-2)
+        next_step, _ = kernel(*initial_state, 1e-2)
         print(next_step[0][1].eval({y_vv: 0}))
```

Comment:
Our examples should probably use `aesara.function` and the returned updates (e.g. to avoid potentially confusing users down the line).